src/networks/blast.ts

```diff
@@ -0,0 +1,90 @@
+import { getProviderForNetwork, proxyRpcEndpoint } from '@/handlers/web3';
+import { Network, NetworkProperties } from './types';
+import { gasUtils } from '@/utils';
+import type { Chain } from '@wagmi/chains';
+// import { blast } from '@wagmi/chains';
+import { getAvalancheGasPrices, getBlastGasPrices } from '@/redux/gas';
+import { getRemoteConfig } from '@/model/remoteConfig';
+import { BLAST_MAINNET_RPC } from 'react-native-dotenv';
+import { BLAST_ETH_ADDRESS } from '@/references';
+
+const BLAST_CHAIN_ID = 81457;
+export const chainBlast: Chain = {
```

Comment:
can we migrate from `wagmi/chains` and use `viem/chains` looks like they moved it over and thats where the updates are being put now then we dont have to define this shiz ourself

Reply:
yeah